packages/stark-ui/src/modules/toast-notification/components/toast-notification.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="stark-toast" [ngClass]="getMessageTypeClass()">
+
+	<mat-icon svgIcon="information" starkSvgViewBox *ngIf="message.type === 0"></mat-icon>
+	<mat-icon svgIcon="alert-circle" starkSvgViewBox *ngIf="message.type === 1"></mat-icon>
+	<mat-icon svgIcon="alert" starkSvgViewBox *ngIf="message.type === 2"></mat-icon>
+
+	<span class="stark-toast-text"
+			role="alert"
+			translate>{{ message.key }}
```

Comment:
You can improve this to allow HTML as part of the message. Just replace this line by this one (I've already tested it 😉 ):
```
[innerHTML]="message.key | translate"
```